src/Tizen.Uix.Tts/Tizen.Uix.Tts/TtsClient.cs

```diff
@@ -203,6 +203,37 @@ public enum Voice
         Child
     };
 
+    /// <summary>
+    /// Enumeration for the states of TTS service.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    public enum ServiceState
+    {
+        /// <summary>
+        /// Ready state.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        Ready = 0,
+
+        /// <summary>
+        /// Synthesizing state.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        Synthesizing = 1,
+
+        /// <summary>
+        /// Playing state.
+        /// </summary>
+        /// <since_tizen> 10 </since_tizen>
+        Playing = 2,
```

Comment:
just wondering - can those state overlap? i.e. parallel synth and stream play in real time?

Reply:
In the aspect of real behavior of TTS service engine, yes it is possible to parallely synth and stream play in some times.
We just want to seperate the timing between the begining of the synthesis and the begining of the playing. That is, `Synthesizing` state means that the TTS service engine only synthesizes the sound data without playing.
TTS service engine will play sound data when the synthesized data is enough to play and then the service state will be changed to `Playing`.